db/schema.rb

```diff
@@ -35,4 +35,11 @@
     t.check_constraint "srid > 0 AND srid <= 998999", name: "spatial_ref_sys_srid_check"
   end
 
+  create_table "types", force: :cascade do |t|
+    t.string "name"
```

Comment:
I think you should validate name and description, so that they don't accept null and a minimum length for both of them.